api/capi/src/nnstreamer-capi-util.c

```diff
@@ -570,8 +570,8 @@ ml_tensors_data_destroy (ml_tensors_data_h data)
 
   _data = (ml_tensors_data_s *) data;
 
-  if (_data->handle != NULL) {
-    status = ml_single_destroy_notify (_data->handle, _data);
+  if (_data->destroy) {
```

Comment:
Q: If checking ```_data->destory``` whether NULL or not, is it OK without the checking for ```_data``` ?

Reply:
It is already checked at line 568.